src/griffe_pydantic/static.py

```diff
@@ -87,6 +89,7 @@ def process_attribute(attr: Attribute, cls: Class, *, processed: set[str]) -> No
         kwargs["default"] = attr.value
 
     if attr.name == "model_config":
+        kwargs.pop("ignored_types", None)  # Ignored types cannot be serialized yet.
```

Comment:
This resolved to an `ExprTuple` which cannot be serialized by to `str`. Here is the Pydantic signature:

```py
class Model(BaseModel)
    model_config = ConfigDict(ignored_types=(cached_property,))
```

Reply:
Can you open a new issue describing the problem, with a reproducible code example?